docs/submission/clinical-data-validation-rules.md

```diff
@@ -37,19 +37,45 @@ These identifier fields allow for linking across the different clinical events a
+
+- The `age_at_primary_diagnosis` is used as the reference time point.
+- The day the patient dies is the clinical endpoint (`survival_time`).
+
+Examples of time interval validation checks:
+
+- If a patient's `vital_status` is `Deceased`, all time intervals must be less than or equal to `survival_time`.
+- The `relapse_interval` must be less than the `interval_of_followup` in the Follow Up the relapse was recorded.
+- If a follow up is associated with a particular treatment (via the `submitter_treatment_id`), the `interval_of_followup` must be greater than the `treatment_start_interval`.
+
 
 ### Donors Older than 90 years old
 
 Since the occurrence of individuals over the age of 90 is rare, it is therefore considered a potentially identifiable value. Thus, the allowed value for the `age_at_diagnosis` field is capped at 90.
 
+## Cross Field Validations
 
+A number of cross-field consistency checks within files are implemented to ensure quality control and data correctness. This involves requiring the value of another field to validate the current field. The cross-field validation checks are implemented using Javascript. For the advanced user, you will be able to see the actual cross-field validation scripts in the Dictionary Viewer by clicking on `View Script`. Examples include:
+
+- Criteria for staging fields are dependent on the selected `clinical_staging_system`.
+- Submitted `tumour_grade` is checked against selected `tumour_grading_system`.
+- Valid values for `specimen_type` are cross-checked with the `tumour_normal_designation` field.
+- The requirement for fields related to relapse/recurrence are dependent on the `disease_status_at_followup` field.
+- The requirement for `survival_time` is depenedent on the `vital_status` field.
 
-## Cross File Validations
 
+## Cross File Validations
 
-(insert diagram and table here)
+Relationships between different clinical fields across files are validated to ensure data correctedness. These include for example:
 
+- A `submitter_sample_id` must belong to only one `submitter_specimen_id`. A `submitter_specimen_id` must belong to only one `submitter_donor_id`.
+- A `submitter_donor_id` or `submitter_specimen_id` submitted in any of the clinical submission files is checked to ensure it exists in the `Sample Registration` file.
+- A `submitter_specimen_id` in the `Specimen` file must belong to a registered `submitter_donor_id` in the `Sample Registration` file.
+- A `submitter_primary_diagnosis_id` in the `Treatment`, `Specimen` or `Follow Up` file must exist in the `Primary Diagnosis` file.
+- A `submitter_treatment_id` in the `Follow Up` file must exist in the `Treatment` file.
```

Comment:
Hardeep, question for you...

When in all these points, when you say "must exist in the Treatment file", for example - do we want to instead say something like "must have been submitted using the Treatment file", or "Must have been submitted in the sample registration file".  

Because the field could "exist" in a file they have prepped but really it's only when they submit the files does the cross validation occur.

could be clear enough, so I'll let you decide.